src/components/User.vue

```diff
@@ -30,7 +30,8 @@ watchEffect(() => {
             size="18"
             class="mr-2"
           />
-          <span class="truncate">{{ username }}</span>
+          <span class="w-[50px] truncate sm:hidden">{{ fullUserName }}</span>
```

Comment:
Well, this is still not optimal.
![image](https://user-images.githubusercontent.com/6792578/155603818-7a699aa4-a70d-4829-a895-682fc657c265.png)
A solution you'll find here in the comments: https://css-tricks.com/flexbox-truncated-text/
But tbh... the issue lies deeper and is simply related to address shortening in general. I tried to solve this before and didn't get to any flawless solution either. I'll merge it as is.